scipy/stats/_morestats.py

```diff
@@ -4023,14 +4023,24 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate', *,
     return res
 
 
-def directionalmean(samples, *, axis=0, normalize=True):
+DirectionalStats = namedtuple('DirectionalStats',
+                              ('directional_mean', 'magnitude'))
+
+
+def directional_stats(samples, *, axis=0, normalize=True):
     """
-    Computes the directional mean of a sample of vectors.
+    Computes the directional mean and magnitude of the mean vector
```

Comment:
Here it says just "mean vector" whereas below it is "mean resultant vector" (as it is in the literature, which is why I suggested the function `mean_resultant` with attributes `direction` and `magnitude`). I'd suggest keeping it consistent (preferably with "resultant").

Reply:
Yes, I will change that to "mean resultant vector".

Reply:
Done.